src/IO/Exporter/PointwiseInterpolator.cpp

```diff
@@ -155,13 +155,7 @@ void interpolate_to_point_impl(
     const tnsr::I<double, Dim, Frame::ElementLogical>& x_element_logical,
     const Mesh<Dim>& mesh, const size_t offset, const size_t length,
     const std::vector<DataVector>& tensor_data) {
-  // Copy into DataVector can be elided by supporting interpolation of double in
-  // intrp::Irregular
-  tnsr::I<DataVector, Dim, Frame::ElementLogical> x_element_logical_dv(1_st);
-  for (size_t d = 0; d < Dim; ++d) {
-    x_element_logical_dv.get(d) = x_element_logical.get(d);
-  }
-  const intrp::Irregular<Dim> interpolant(mesh, x_element_logical_dv);
+  const intrp::Irregular<Dim> interpolant(mesh, x_element_logical);
```

Comment:
Yes, this elides it, but why can't you make two commits: 1. add support for double values to Interpolator 2. use ElementSearchTree in Interpolator

I'll review once the commits are organized better.